### PR TITLE
PWN-3511 - Libs version update

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3511

## Description of Work

PWN-3511 - kotlinVersion - 1.6.10 -> 1.6.21, build:gradle - 7.1.2 -> 7.1.3, roomVersion - 2.4.1 -> 2.4.2, org.robolectric:robolectric - 4.5.1 -> 4.6.1

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -242,7 +242,7 @@ dependencies {
     // https://mvnrepository.com/artifact/junit/junit
     testImplementation 'junit:junit:4.13.2'
     // https://github.com/robolectric/robolectric
-    testImplementation "org.robolectric:robolectric:4.5.1"
+    testImplementation "org.robolectric:robolectric:4.6.1"
```
Comment (1sakov) [inline app/build.gradle:245]: Don't do it, tests will fail. 
It's bugged version, all devs are using 4.5.1

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (kabramovp2p) [reply app/build.gradle:245]: ohh I see 
ok will revert it 👌🏻👍🏻

Comment (gslevinkov) [reply app/build.gradle:245]: why do we need robolectric:robolectric at first place?

**Review by gslevinkov — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (gslevinkov) [reply app/build.gradle:245]: I meant do we use roboelectic in our current unit tests?

Comment (kabramovp2p) [reply app/build.gradle:245]: @1sakov ^^ ?

Comment (kabramovp2p) [reply app/build.gradle]: @1sakov fixed! ^^

Comment (1sakov) [reply app/build.gradle]: we do

**Review by 1sakov — APPROVED** [review]

**Review by kuznetsov — APPROVED** [review]

**Review by davran1001 — APPROVED** [review]

**Pull request merged**